### ISSUE TITLE
Curious about the Batch-related Convolutional Cell.

### ISSUE THREAD

**Issue by miracle24** [ZJULearning/PTL#2]
Hello. I just read your paper, which is great, obtaining great performance in several benchmarks. Therefore, I am curious about the proposed batch-related convolutional cell. In my view, in the BConv cell, a state vector is maintained and updated with each batch. What is strange to me is that, if the state vector is related to each batch instead of each image in the batch, then one image will have different features if you put it in different batch. Therefore, how to determine the feature of the image during retrieval? Maybe I misunderstood the BConv cell. Please help me figure it out!
Thanks a lot.

**Comment by zhengxuyu** [comment]
> Hello. I just read your paper, which is great, obtaining great performance in several benchmarks. Therefore, I am curious about the proposed batch-related convolutional cell. In my view, in the BConv cell, a state vector is maintained and updated with each batch. What is strange to me is that, if the state vector is related to each batch instead of each image in the batch, then one image will have different features if you put it in different batch. Therefore, how to determine the feature of the image during retrieval? Maybe I misunderstood the BConv cell. Please help me figure it out!
> Thanks a lot.

Hi,
Thanks for asking. 
We offer a stable-learning based explanation to help understand how BConv-Cells work. The data generation bias can result in substantial distribution variations between mini-batches. Consequently, models trained in such dataset will fall into the statistical correlation of features and result in unstable performance to yielding lower training loss. To mitigate this problem, the BConv-Cells progressively aggregates the global information of the training dataset and then applying it to mitigate the impacts of data generation bias during training.
